tests/models/perceiver/test_modeling_perceiver.py

```diff
@@ -903,7 +903,7 @@ def test_inference_image_classification(self):
         expected_shape = torch.Size((1, model.config.num_labels))
         self.assertEqual(logits.shape, expected_shape)
 
-        expected_slice = torch.tensor([-1.1653, -0.1993, -0.7521], device=torch_device)
```

Comment:
In this case I updated the test values. I considered this OK as other tests that were previously failing now pass with and the difference was 0.0001 across all elements. 

I wasn't able to track down exactly where this is introduced. LMK if you think this is an issue and I can do some more digging.

Reply:
With a VM as close as to the runner, I get 

```python
tensor([-1.1652, -0.1993, -0.7521], device='cuda:0')
```

and with numpy, it is
```
array([-1.1652362 , -0.19931625, -0.75210035], dtype=float32)
```

Let me check within docker ...

Reply:
On the runner within docker

```bash
(Pdb) logits[0, :3]
tensor([-1.1652, -0.1992, -0.7520], device='cuda:0')

(Pdb) logits[0, :3].to("cpu").numpy()
array([-1.1652446 , -0.19924526, -0.7520099 ], dtype=float32)
``` 

maybe let's use this value and see if we get lucky in the next CI?

Reply:
OK, sounds good. Just to confirm - I'll update the values to the ones when running within docker: 
`[-1.1652446 , -0.19924526, -0.7520099 ]`

Reply:
Yes

Reply:
I've updated them and pushed. Once all the tests have passed, I'll merge and keep an eye out for the slow run CI tests if it needs to be updated again. 

Thanks for your help in looking into these numbers.